File: staylio_backend/src/main/java/com/example/staylio_backend/dto/response/HotelResponse.java
package com.example.staylio_backend.dto.response;

import com.example.staylio_backend.model.enums.HotelStatus;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class HotelResponse {
    private Long id;
    private String name;
    private String description;
    private String imageUrl;
    private HotelStatus status;
    private String hostHotelName;
    private boolean isActive;
    private Integer branchCount;
}

File: staylio_frontend/common/interfaces/response/HotelResponse.ts
import { HotelStatus } from "@common/enums/HotelStatus";

export interface HotelResponse {
    id: number;
    name: string;
    description: string;
    imageUrl: string;
    policy?: string;
    hostHotelName: string;
    status: HotelStatus;
    active: boolean;
    branchCount?: number;
}